virttest/utils_conn.py

```diff
@@ -1426,9 +1425,15 @@ def client_setup(self):
         """
         # initialize variables
         tmp_dir = self.tmp_dir
+        scp_new_cacert = self.scp_new_cacert
+        # sometimes, need to reuse previous CA cert
+        if self.ca_cakey_path and scp_new_cacert == 'no':
```

Comment:
May I ask why we need to check if scp_new_cacert is 'no' here?

Reply:
If scp_new_cacert = 'yes', then it means DONOT reuse previous CA cert.

In build_CA(), when ca_cakey_path is passed in parameter, it will create new CA cert using previous cakey.

Reply:
Okay, but in L1459~1473, there is no checks about scp_new_cacert's value, is that correct?

Reply:
I missed that part, I will update it right now

Reply:
Done